css/style.css

```diff
@@ -381,6 +381,12 @@ video {
     transition-duration: 0.4s;
   }
 
+  .button--sm {
```

Comment:
are you sure this was the intent of the task?

Reply:
no, but is an easy exit :D, I need talk with Jose about that